conformance/operator_test.go

```diff
@@ -355,8 +355,11 @@ var _ = Describe("operator", func() {
 			BeforeEach(func() {
 				var err error
 				node := sriovInfos.Nodes[0]
-				intf, err = sriovInfos.FindOneSriovDevice(node)
-				Expect(err).ToNot(HaveOccurred())
+				intf, err = sriovInfos.FindOneMellanoxSriovDevice(node)
```

Comment:
Can you just add a comment explaining why we are doing this? The skip part is pretty far, and reading here it's not clear why we are preferring mellanox cards to other models.